packages/backend/src/clients/Slack/SlackStorage.ts

```diff
@@ -1,15 +1,13 @@
+import { Installation, InstallationQuery } from '@slack/bolt';
```

Comment:
Would be nice in a future PR to convert this into a model class.

Reply:
Created a new issue for this :point_up: